response_operations_ui/forms.py

```diff
@@ -45,6 +45,30 @@ class SecureMessageForm(FlaskForm):
     hidden_to_ru_id = HiddenField('hidden_to_ru_id')
 
 
+class RespondentSearchForm(FlaskForm):
+    first_name = StringField('first_name')
+    last_name = StringField('last_name')
+    email_address = StringField('email_address')
+    source = HiddenField('source')
+    page = HiddenField('page')
+    submit = SubmitField('Search')
+
+    def validate(form):
+        first_name = form.first_name.data or ''
+        last_name = form.last_name.data or ''
+        email_address = form.email_address.data or ''
+        source = form.source.data or 'home'
+
+        if not (len(first_name) > 0 or len(last_name) > 0 or len(email_address) > 0):
```

Comment:
Can this be rewritten `if not first_name or last_name or email_address:` I think empty values are false

Reply:
Done in the next commit